Cargo.toml

```diff
@@ -46,7 +46,10 @@ warp = "0.3.2"
 [features]
 basic-auth = ["base64"]
 charsets = ["encoding_rs", "encoding_rs_io"]
-compress = ["flate2"]
+# The following three compress features are mutually exclusive.
+compress = ["flate2/default"]
+compress-zlib = ["flate2/zlib"]
+compress-zlib-ng = ["flate2/zlib-ng"]
```

Comment:
I think the features a also listed in lib.rs, can you update that too?

Reply:
Done and all tests are green.